src/metrics/include/metrics/metrics.hpp

```diff
@@ -595,54 +70,35 @@ class MetricsClockSource {
  */
 template <typename ClockSource>
 class RegistryT {
+private:
+    using EventSeries = typename v1::OperationImpl<ClockSource>::EventSeries;
+    using OperationsMap = std::unordered_map<
+        std::string,
+        std::unordered_map<std::string, std::unordered_map<ActorId, EventSeries>>>;
```

Comment:
Type seems right but pretty dense. Let's add comments about what each of these sub-fields means:

```c++
using OperationsMap = std::unordered_map<
  std::string, // actor name
  std::unordered_map<std::string, // operation name
                     std::unordered_map<ActorId, EventSeries> // the series for each instance of the Actor
  >>
```

or create simpler `using`s for the sub-fields e.g.
```c++
using SeriesPerActorId = std::unordered_map<ActorId, EventSeries>>;
```
and then use those so you end up with something like
```c++
using OperationsMap = std::ordered_map<std::string, OperationsForActor>
```

Reply:
I added a couple `using` declarations but went with the names `OperationsByThread` and `OperationsByType` to match the local variables I declared in `v1::RegistryT::operation()`.